src/index.js

```diff
@@ -1,13 +1,54 @@
 const _ = require('lodash');
 
-const { startServer } = require('./server');
-const { blockchainEnv } = require('./constants');
-const { getDevQtumExecPath } = require('./utils');
+const BodhiServer = require('./server');
```

Comment:
can we name these as camelCase? Typically we reserve PascalCase for classes

Reply:
@alex-cory Isn't this considered a bare object?
https://github.com/airbnb/javascript#naming--PascalCase-singleton